src/app/_components/MobileNavigation.tsx

```diff
@@ -0,0 +1,151 @@
+'use client'
+
+import { Fragment, useState, Dispatch, SetStateAction } from 'react'
+
+import Link from 'next/link'
+
+import { List, X, Icon as IconType } from '@phosphor-icons/react'
+import clsx from 'clsx'
+import { Route } from 'next'
+
+import { Icon } from '@/components/Icon'
+import { Logo } from '@/components/Logo'
+import { SlideOver } from '@/components/SlideOver'
+import { Social } from '@/components/Social'
+import { baseStyles } from '@/components/TextLink'
+
+import { PATHS } from '@/constants/paths'
+
+const getInvolvedItems = [PATHS.EVENTS, PATHS.GRANTS]
+const communityItems = [PATHS.ECOSYSTEM, PATHS.GOVERNANCE]
+
+type MobileLinkProps = {
+  label: string
+  path: Route
+  nested?: boolean
+  setOpen: Dispatch<SetStateAction<boolean>>
+}
+
+function MobileLink({ label, path, nested, setOpen }: MobileLinkProps) {
+  const touchTargetSpacing = 5
+  const touchTargetClass = `px-${touchTargetSpacing}`
+  const touchTargetOffsetClass = `-ml-${touchTargetSpacing}`
+
+  return (
+    <li
+      className={clsx(
+        nested ? 'border-l border-brand-400' : touchTargetOffsetClass,
+      )}
+    >
+      <Link
+        href={path}
+        className={clsx(
+          baseStyles,
+          touchTargetClass,
+          'inline-block px-5 py-2.5',
+        )}
+        onClick={() => setOpen(false)}
+      >
+        {label}
+      </Link>
+    </li>
+  )
+}
+
+type IconButtonProps = {
+  icon: IconType
```

Comment:
What about using `IconProps['component']` - We could also refactor it into a type and export it `  icon?: IconProps['component']` - Since we already have this centralized?

Reply:
Yes I like that! Let's derive from `IconProps` for now :)